playbooks/roles/ocp-cluster-logging/tasks/main.yml

```
@@ -45,6 +52,11 @@
 
   - name: Run ImageContentSourcePolicy
     shell: oc apply -f "{{ role_path }}/files/ImageContentSourcePolicy.yml"
+    when: ocp_version | float < 4.13
+
+  - name: Create ImageDigestMirrorSet
+    shell: "{{ role_path }}/files/ImageDigestMirrorSet.sh {{ logging_version }}"
+    when: ocp_version | float > 4.12
```

```suggestion
    when: ocp_version | float >= 4.13
```
Can we have like this?